plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -47,6 +47,14 @@
       - Specify the password to authenticate with.
       - You can use C(PROXMOX_PASSWORD) environment variable.
     type: str
+  api_token_id:
+    description:
+      - Specify the token ID.
+    type: str
```

Comment:
```suggestion
    type: str
    version_added: 1.3.0
```
Same for the other.

I guess it's time for a doc fragment soon, and maybe some shared module_utils which provides the argspec for the doc fragment, and handles things like creating the `ProxmoxAPI` client? :)

Reply:
Good idea, I will look into it.

Reply:
Should I include the doc fragment in this PR or in another one ?

Reply:
Both is fine for me.

Reply:
Ok, I will work on this in an other PR, probably next week.